modules/ROOT/partials/integrations/react-tech-ref.adoc

```diff
@@ -504,9 +504,9 @@ Type : `+String+`
 
 Only valid when the editor is in classic (iframe) mode. Sets the `+name+` attribute for the `+textarea+` element used for the editor in forms.
 
-Default value : `+undefined+`
+*Default value:* `+'undefined'k+`
```

Comment:
Agreed though I'm wondering if we should log a separate Jira to clean up these integration options? as they generally are inconsistent since the `Type` isn't first either.

Reply:
It is better to log another Jira ticket. We might need a separate ticket to fix up the markups of the content of `Type`, `Possible value` and `Default value`